code/game/objects/items/robot/robot_parts.dm

```diff
@@ -62,6 +62,81 @@
 				return 1
 	return 0
 
+/obj/item/robot_suit/wrench_act(mob/living/user, obj/item/I) //Deconstucts empty borg shell. Flashes remain unbroken because they haven't been used yet
+	var/turf/T = get_turf(src)
+	forceMove(T)

```

Comment:
why are you moving the shell to the turf as well?

Reply:
Originally, I wanted to make sure all the parts ended up at the same spot, so it would pull the naked shell out of the player's hand (if they were holding it) and drop it on the ground. But there's probably no real reason to force this, so I'll remove that line.